modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request-organization.json

```diff
@@ -8,14 +8,16 @@
 		},
 		"permissions": {
 		},
+		"publish": {
+		},
 		"update": {
 		}
 	},
-	"active": true,
+	"active": false,
 	"label": {
-		"en_US": "EVP Organization"
+		"en_US": "EVP Request Organization"
```

Comment:
I have a question at this point. Should it be "EVP Request Organization" or "EVP Organization Request"?